ui/helpers/constants/design-system.ts

```diff
@@ -36,7 +36,6 @@ export enum Color {
   infoDefault = 'info-default',
   infoMuted = 'info-muted',
   infoInverse = 'info-inverse',
-  mainnet = 'mainnet',
```

Comment:
Removing from `Color` enum as it is not used 

![Screenshot 2025-02-07 at 3 01 12 PM](https://github.com/user-attachments/assets/4e2cd3f8-1d5b-4002-b6ce-d746c83a8b0c)